tests/e2e/pageobjects/ide/OpenEditors.ts

```diff
@@ -0,0 +1,63 @@
+/*********************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import 'reflect-metadata';
+import { injectable, inject } from 'inversify';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { CLASSES } from '../../inversify.types';
+import { By } from 'selenium-webdriver';
+import { Logger } from '../../utils/Logger';
+import { TimeoutConstants } from '../../TimeoutConstants';
+
+@injectable()
+export class OpenEditors {
```

Comment:
ID is more reliable compare to css classes, because is usually permanent and so preferable to reference in locators:
![Screenshot from 2022-01-13 12-24-37](https://user-images.githubusercontent.com/1197777/149312522-b87b6cd4-0d78-4756-a309-5266dbdb660a.png)